### PR TITLE
afl-plus-plus 4.10c (new formula)

### PR DESCRIPTION
**Pull request by pthariensflame**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

### CODE REVIEW

**Comment by pthariensflame** [comment]
How do I resolve this audit?

**Comment by pthariensflame** [comment]
Kind of stumped now.

**Comment by SMillerDev** [comment]
Do you know where the symbol should come from ?
```console
  cc1: error: cannot load plugin ./afl-gcc-pass.so: ./afl-gcc-pass.so: undefined symbol: _ZNK6frange6acceptERK14vrange_visitor

```

**Comment by pthariensflame** [comment]
Let's see if this works?

**Comment by pthariensflame** [comment]
Ah, can this be marked as "in-progress"?
I'll loop back to this in a week or less.

**Comment by pthariensflame** [comment]
This is extremely frustrating.  It builds, tests, and audits perfectly cleanly on my local machine, and yet the CI machine that's supposed to be that exact same environment isn't happy.  Why is audit complaining here and not for me?  Why does the Linux build fail *during fetch*?

**Comment by SMillerDev** [comment]
Do you use docker locally for this?

**Comment by pthariensflame** [comment]
> Do you use docker locally for this?

No, I build it in macOS.

**Comment by SMillerDev** [comment]
Somehow I thought it was about Linux. For macOS it seems weird it's building `.so` libraries to begin with. Maybe there is some configuration that needs to be told it's on macOS?

File: Formula/a/afl-plus-plus.rb [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+  depends_on "llvm"
+  depends_on "python@3.12"
+
+  conflicts_with "afl-fuzz", because: "they are different versions of the same software"
+
+  # Fix incompatibility with BSD coreutils.  Also, we don't want to
+  # alter the testing environment, and test_shm fails on macOS by
+  # default and hinders installation of working stuff so just remove
+  # that check entirely.
+  patch :DATA
```
Comment (fxcoudert) [inline Formula/a/afl-plus-plus.rb:21]: This patch should be submitted, reviewed and (at least) accepted by upstream before we ship it.

File: Formula/a/afl-plus-plus.rb [hunk 2]
```diff
@@ -0,0 +1,218 @@
+class AflPlusPlus < Formula
+  desc "American fuzzy lop plus plus: Security-oriented fuzzer, community-maintained"
+  homepage "https://github.com/AFLplusplus/AFLplusplus"
+  url "https://github.com/AFLplusplus/AFLplusplus.git",
+      tag:      "v4.10c",
+      revision: "775861ea94d00672c9e868db329073afd699b994"
```
Comment (fxcoudert) [inline Formula/a/afl-plus-plus.rb:6]: Could it build from a tarball instead?

**Review by fxcoudert — CHANGES_REQUESTED** [review]

**Pull request closed without merging**